AdjustableForKeyboard.swift

```diff
@@ -1,11 +1,11 @@
-import Foundation
+
 import UIKit
 
-protocol AdjustableForKeyboard {
+protocol AdjustableForKeyboard: class {
```

Comment:
Why class here?

Reply:
Since you're using this explicitly on UIViewControllers it makes sense to restrict this protocol to classes.